docs/content/users/install/ddev-installation.md

```diff
@@ -97,57 +97,78 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
+    * WSL2 is not the same as Docker Desktop’s WSL2 engine.  
+      Using WSL2 to install and run DDEV is not the same as using Docker Desktop’s WSL2 engine, which itself runs in WSL2, but can serve applications running in both traditional Windows and inside WSL2.
 
     The WSL2 install process involves:
 
     * Installing Chocolatey package manager (optional).
     * One time initialization of mkcert.
     * Installing WSL2 and installing a distro like Ubuntu.
-    * Installing Docker Desktop for Windows and enabling WSL2 integration with the distro (optional, do this if you're using the Docker Desktop approach).
-    * Installing DDEV inside your distro.
+    * Optionally installing Docker Desktop for Windows and enabling WSL2 integration with the distro (if you're using the Docker Desktop approach).
+    * Installing DDEV inside your distro; this is normally done by running one of the two scripts below, but can be done manually step-by-step as well.
 
     ### WSL2 + Docker CE Inside Install Script
 
-    This scripted installation prepares your default WSL2 Ubuntu distro and has no dependency on Docker Desktop.
+    This scripted installation prepares your default WSL2 Ubuntu distro and has no dependency on Docker Desktop. It is designed to be able to run multiple times without breaking anything.
+
+    The provided PowerShell script can do most of the work for you, or you can handle these things manually. (This script works with the built-in PowerShell v5, but not with the newer v7.)
 
-    The provided `PowerShell.exe` (original PowerShell 5) script can do most of the work for you, or you can handle these things manually. 
     In all cases:
     
-    1. Install WSL2 with an Ubuntu distro. On a system without WSL2, run:
-        ```powershell
-        wsl --install
-        ```
+    1. Install WSL2 with an Ubuntu distro.
+
+        * Install WSL with 
+            ```
+            wsl --install`
+            ```
+
+        * Reboot if required. (Usually required.)
 
-        Verify that you have an Ubuntu distro set to default by running `wsl -l -v`.
+        * Visit the Microsoft Store and install the *updated* "Windows Subsystem for Linux" and click "Open". It will likely prompt you for a username and password for the Ubuntu WSL2 instance it creates.
```

Comment:
```suggestion
        * Visit the Microsoft Store and install the *updated* ["Windows Subsystem for Linux"](https://apps.microsoft.com/store/detail/windows-subsystem-for-linux/9P9TQF7MRM4R) and click "Open". It will likely prompt you for a username and password for the Ubuntu WSL2 instance it creates.
```